rerun_py/rerun_sdk/rerun/notebook.py

```diff
@@ -287,41 +293,66 @@ def set_active_recording(
 
         self._viewer.set_active_recording(recording_id)
 
+    @deprecated_param("nanoseconds", use_instead="duration or timestamp", since="0.23.0")
+    @deprecated_param("seconds", use_instead="duration or timestamp", since="0.23.0")
     def set_time_ctrl(
         self,
         *,
         sequence: int | None = None,
-        nanoseconds: int | None = None,
-        seconds: float | None = None,
+        duration: int | float | timedelta | np.timedelta64 | None = None,
+        timestamp: int | float | datetime | np.datetime64 | None = None,
```

Comment:
@abey79 should we maybe introduce type aliases for this? `DurationLike` or whatever?